### PR TITLE
#94 Realease notes for v1.1.0

### PR DESCRIPTION
**Pull request by albertochimentiinbibo**
This PR adds the release notes for v1.1.0 of Shift.

You can have a look at the list of contributions taking part to the release in the related issue.

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+
+#### Integration & Resources
+
+- Nuke: Menu file to setup Shift tools inside Nuke.
+- Nuke (Beta): "ShiftWorkflow" Nuke node to execute Shift workflow inside the Nuke node graph.
+
+#### API
+
+- Added `shift.utils.dcc_threading` module containing utilities to handle threaded execution in DCC context
+- Added `shift.utils.decorators` module with `mainThread` decorator for SOperator.execute methods to force execution on main thred in Maya and Nuke
```
Comment (hectorjimenezinbibo) [inline docs/release_notes.md:24]: Remember to add the full stop at the end of the sentence to be aligned with the rest.
```suggestion
- Added `shift.utils.dcc_threading` module containing utilities to handle threaded execution in DCC context.
- Added `shift.utils.decorators` module with `mainThread` decorator for SOperator.execute methods to force execution on main thred in Maya and Nuke.
```

**Review by hectorjimenezinbibo — CHANGES_REQUESTED** [review]
Minor details 👍

**Review by juliavisserinbibo — CHANGES_REQUESTED** [review]

File: docs/release_notes.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+- `SHIFT_SINGLE_THREADED` environment variable to switch the threaded execution behaviour.
+- Workflow execution progress bar in the Shift UI.
+- Board styling for higlighting the execution status of the workflow.
+- Ability to request the interruption of an executing workflow with the "Clear Execution" button. The button will now clear the workflow only if pressed while no execution is running on the current board.
+- Ability to rename a node by double clicking on its name with the left mouse button.
+
+#### Integration & Resources
+
+- Nuke: Menu file to setup Shift tools inside Nuke.
+- Nuke (Beta): "ShiftWorkflow" Nuke node to execute Shift workflow inside the Nuke node graph.
```
[the remark this answers is not in the document]
Comment (albertochimentiinbibo) [reply docs/release_notes.md:19]: What do you think about this?
```suggestion
**Nuke**
- Added a Menu file to set up Shift tools inside Nuke.
- (Beta) Included a new "ShiftWorkflow" Nuke node to execute Shift workflow inside the Nuke node graph. This node is in beta, meaning that its behaviour might change in the future.
```

File: docs/release_notes.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+
+- Added `shift.utils.dcc_threading` module containing utilities to handle threaded execution in DCC context.
+- Added `shift.utils.decorators` module with `mainThread` decorator for SOperator.execute methods to force execution on main thred in Maya and Nuke.
+- Added `disableEditing` method to `ParameterWidget` class to enable/disable editing capabilities of the Inspector widgets.
+
+### Improvements
+
+#### Core
+
+- Set Focus Execution to `False` by default. The global execution status is now tracked on the progress bar. *Shift-1218*
```
[the remark this answers is not in the document]
Comment (albertochimentiinbibo) [reply docs/release_notes.md:31]: I thought it should have been for imporvements as well. Ok!

Committing the suggestion

File: docs/release_notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/release_notes.md:19]: Good

File: docs/release_notes.md [hunk 4]
```diff
@@ -1,5 +1,67 @@
...
+
+#### Catalogs
+
+**Standard Catalog (Version 1.1.0)**
+- Added `mainThread` boolean plug to PythonScript node to force execution on main thread in DCC context. This is needed to use DCC Python API calls for Maya or Nuke in the node's script. *Shift-1218*
```
[the remark this answers is not in the document]
Comment (albertochimentiinbibo) [reply docs/release_notes.md:36]: This logic is specific for DCC context for Maya and Nuke. I would not say of an application and I would avoid the "for example" as those are the only cases in which it has effect.

File: docs/release_notes.md [hunk 5]
```diff
@@ -1,5 +1,67 @@
...
+
+**CreativeSoftware Catalog (Version 1.1.0)**
+- This catalog's nodes have been made compatible with threaded execution feature in Maya. *Shift-1218*
```
[the remark this answers is not in the document]
Comment (albertochimentiinbibo) [reply docs/release_notes.md:39]: What do you think about:

```suggestion
- This catalog's nodes have been made compatible with the new threaded execution feature in Maya and Nuke. *Shift-1218*
```

All node's logic has been wrapped so it does not make sense to say that only the ones for Maya and Nuke have been modified.

File: docs/release_notes.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/release_notes.md:37]: Ok, leave it as it is then

File: docs/release_notes.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/release_notes.md:40]: Is just that the way it is worded sounds that the threaded execution is a change in Maya and Nuke

File: docs/release_notes.md [hunk 6]
```diff
@@ -1,5 +1,69 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Added
+
+#### Core
+
+- Shift workflows now run on a dedicated thread, enabling users to inspect and author other workflows and interact with the UI while there is an ongoing execution.
+- An environment variable, `SHIFT_SINGLE_THREADED`, has been added to enable or disable threaded execution.
```
Comment (hectorjimenezinbibo) [inline docs/release_notes.md:10]: Question, here we're mentioning the use of the environment variable, should we also mention the option in the menu to change it on-the-fly?

Comment (albertochimentiinbibo) [reply docs/release_notes.md:10]: yes, let me do it.

Comment (albertochimentiinbibo) [reply docs/release_notes.md:11]: done

File: docs/release_notes.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (albertochimentiinbibo) [reply docs/release_notes.md:41]: What about

```suggestion
- This catalog's nodes have been made compatible with threaded execution feature. *Shift-1218*
```

[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/release_notes.md:41]: This is good

**Review by juliavisserinbibo — APPROVED** [review]

**Review by hectorjimenezinbibo — APPROVED** [review]

**Pull request merged**